Servers/WindowServer/WSWindowManager.cpp

```diff
@@ -961,6 +964,10 @@ void WSWindowManager::process_mouse_event(WSMouseEvent& event, WSWindow*& hovere
             event_window_with_frame = &window;
             return IterationDecision::Break;
         });
+
+        // Clicked outside of any window
+        if (!mouse_event_in_window && event.type() == WSEvent::MouseDown)
+            set_no_active_window();
```

Comment:
```suggestion
            set_active_window(nullptr);
```

Reply:
Ah okay - tried that at first but it crashed stuff. Must have just been a bug with the function! Fixed now though :)